src/Screens/SelectTableScreen/selectTableScreen.tsx

```diff
@@ -0,0 +1,48 @@
+import React, { useState } from "react";
+import { View } from "react-native";
+import DropDownPicker from "react-native-dropdown-picker";
+import { Button, Text } from "react-native-elements";
+
+import { selectTableStyles } from "../../styles/selectTableScreenStyles";
+
+export const selectTableScreen = ({ navigation, route }) => {
+  const [open, setOpen] = useState(false);
+  const [value, setValue] = useState(route.params?.table || "двоих");
+  const [items, setItems] = useState([
+    { label: "двоих", value: "двоих" },
+    { label: "четверых", value: "четверых" },
+    { label: "шестерых", value: "шестерых" },
+    { label: "восьмерых", value: "восьмерых" },
+    { label: "десятерых", value: "десятерых" },
+  ]);
```

Comment:
If it's const data - don't use state for that

Reply:
![image](https://user-images.githubusercontent.com/53368949/142005651-68d3e9f0-588f-4059-bb2c-5d011956f9e3.png)
 i use setItems for dropDownPicker

Reply:
ok